.gitlab-ci.yml

```diff
@@ -1,13 +1,24 @@
 stages:
+  - containers
   - build
   - test
   - certification-dummy
   - certification-netgear-rax40
   - upload
 
+images-build:
+  stage: containers
+  tags:
+    - shell
+    - docker-build
+  script:
+    - docker login -u $CI_REGISTRY_USER -p $CI_REGISTRY_PASSWORD $CI_REGISTRY
+    - ./tools/docker/image-build.sh -i builder -t $CI_PIPELINE_ID
+    - docker push $CI_REGISTRY_IMAGE/prplmesh-builder:$CI_PIPELINE_ID
```

Comment:
If the build is not tagged as `latest` how will the clients know to pull the latest version?

Reply:
For now, I'm assuming latest will be pushed manually by a developer when the image changes.